library-best-practices/JLBP-11.md

```diff
@@ -0,0 +1,12 @@
+[JLBP-11] Stay up to date with compatible dependencies
```

Comment:
Is "compatible" needed in the title here? What's an incompatible dependency?

Reply:
This is to indicate that it's not a requirement to jump to new major versions in a short time frame.